src/frontend/next/src/components/SnackBar.tsx

```diff
@@ -5,10 +5,11 @@ import CloseIcon from '@material-ui/icons/Close';
 
 type SnackbarProps = {
   message: string;
+  shouldOpen: boolean;
 };
 
-const SimpleSnackbar = ({ message }: SnackbarProps) => {
-  const [open, setOpen] = useState<boolean>(true);
+const SimpleSnackbar = ({ message, shouldOpen }: SnackbarProps) => {
+  const [open, setOpen] = useState<boolean>(shouldOpen);
```

Comment:
What does `shouldOpen` mean? If we're talking about state and it is a boolean, I think `isOpen` or `isClosed` are better choices here